app/views/movements.scala.html

```diff
@@ -36,18 +36,16 @@
                 <th class="table-cell" id="submissionType">@messages("submissions.submissionType")</th>
                 <th class="table-cell" id="submissionAction">@messages("submissions.submissionAction")</th>
                 <th class="table-cell" id="dateUpdated">@messages("submissions.dateUpdated")</th>
-                <th class="table-cell" id="status">@messages("submissions.status")</th>
                 <th class="table-cell" id="noOfNotifications">@messages("submissions.noOfNotifications")</th>
             </tr>
 
             @for( (submission, notifications) <- submissions ){
                 <tr class="table-row">
-                    <td class="table-cell" id="ucr-@submission.conversationId">@submission.ucr</td>
-                    <td class="table-cell" id="submissionType-@submission.conversationId">@submission.submissionType</td>
-                    <td class="table-cell" id="submissionAction-@submission.conversationId">@submission.submissionAction</td>
+                    <td class="table-cell" id="ucr-@submission.conversationId">@submission.ucrBlocks.headOption.map(_.ucr).getOrElse("-")</td>
```

Comment:
And what with other ucr's blocks ?

Reply:
Thanks for spotting this. Don't know if that will ever be the case, but added fix for that.